src/main/java/io/kestra/plugin/jira/issues/JiraClient.java

```diff
@@ -30,18 +34,25 @@ public abstract class JiraClient extends Task implements RunnableTask<VoidOutput
     protected String baseUrl;
 
     @Schema(
-        title = "Atlassian Username"
+        title = "Atlassian Username",
+        description = "(Required for basic & API token authorization)"
     )
     @PluginProperty(dynamic = true)
-    @NotBlank
     protected String username;
 
     @Schema(
-        title = "Atlassian password"
+        title = "Atlassian password or API token",
+        description = "(Required for basic & API token authorization)"
     )
     @PluginProperty(dynamic = true)
-    @NotBlank
-    protected String password;
+    protected String passwordOrToken;
```

Comment:
Thsi plugin has already been published in 0.18 so you cannot change a property name for backward compatibility